snap/snapcraft.yaml

```diff
@@ -12,7 +12,7 @@ confinement: classic
 
 apps:
   snapcraft:
-    command: bin/snapcraft-classic
+    command: $SNAP/usr/bin/python3 $SNAP/bin/snapcraft
```

Comment:
Travis does like this:

> FileNotFoundError: [Errno 2] No such file or directory: '/root/snapcraft/prime/$SNAP/usr/bin/python3'

Reply:
meh, mvo told me it would work, but from the looks of it, we are the ones messing up